docs/reference/query-dsl/query-string-syntax.asciidoc

```diff
@@ -24,7 +24,7 @@ search terms, but it is possible to specify other fields in the query syntax:
     status:active
 
 * where the `title` field contains `quick` or `brown`.
-  If you omit the OR operator the default operator will be used
+  If you omit the OR operator, the default operator will be used (except if `title` field is typed `keyword`)
```

Comment:
I think we should remove this sentence completely. We don't split on whitespace anymore so we shouldn't assume that `quick` and `brown` are going to be separated. It depends only on the tokenizer that is use for the field. We've added a warning section at the beginning of the docs regarding `keyword` field so I think it's enough.